active/NumberBox/NumberBox.md

```diff
@@ -159,24 +185,31 @@ unsealed runtimeclass NumberBox : Windows.UI.Xaml.Controls.Control
     String Text; 
     String PlaceholderText;
 
+<!-- todo: Why does this say "basic"? Is there also an advanced validation mode? -->   
     NumberBoxBasicValidationMode BasicValidationMode;
 
     Boolean AcceptsCalculation;
 
     NumberBoxSpinButtonPlacementMode SpinButtonPlacementMode{ get; set; };
 
+<!-- todo: This should have the "Is" prefix, to differentiate them from events. -->
     Boolean HyperScrollEnabled;
     Boolean HyperDragEnabled;
     Boolean WrapEnabled;
 
+<!-- NumberFormatter is worth an example -->
     Windows.Globalization.NumberFormatting.INumberFormatter2 NumberFormatter;
 
+<!-- todo: Suggest DecrementButtonIcon naming, to match DecrementButtonStyle -->
+
     IconSource DecrementIcon;
     IconSource IncrementIcon;
     Windows.UI.Xaml.Style DecrementButtonStyle;
     Windows.UI.Xaml.Style IncrementButtonStyle;
     NumberBoxTemplateSettings TemplateSettings{ get; };
```

Comment:
Current control does not have properties for icons or styles (there is no reason, if we wanted them to be customizable we should use lightweight styling and define them in resources). I also do not have a TemplateSettings.

Reply:
I believe you're right. I have removed these properties.